jwst/extract_1d/extract.py

```diff
@@ -2887,7 +2887,7 @@ def do_extract1d(
                     log.info("Spectral order 0 is a direct image, skipping ...")
                     continue
 
-                log.info(f'Processing spectral order {sp_order}')
+                log.debug(f'Processing spectral order {sp_order}')
```

Comment:
What's the order of looping within the overall extract_1d process, when you have multiple sources, multiple integrations, and multiple orders? I like the idea of seeing the looping over spectral orders, but perhaps not if it's repeated for every integration.